rust/src/crypto.rs

```diff
@@ -0,0 +1,265 @@
+use derive_more::{AsMut, AsRef, From};
+use sodiumoxide::crypto::{box_, sealedbox, sign};
+
+pub trait ByteObject: Sized {
+    /// Create a new object with all the bytes initialized to `0`.
+    fn null() -> Self;
```

Comment:
i would avoid the use of `null()` as a function name here, because i immediately associated it with `None`. maybe `init()` / `new()` / `zeroed()` suits better.

Reply:
I like `zeroed()`, I'll change that